common/src/deployment.rs

```diff
@@ -26,3 +27,18 @@ pub enum Environment {
     Local,
     Production,
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use crate::deployment::State;
+
+    #[test]
+    fn test_state_deser() {
+        println!("bla");
```

Comment:
```suggestion
```

Reply:
Will do it soon after the unstable deployment finishes. Pushing again it's gonna stop the existing job.